### DISCUSSION TITLE
Improvements to CIECAM02.

### DISCUSSION THREAD

**Discussion by nschloe** [colour-science/colour#709] [Ideas]
I recently implemented CIECAM02 (and CAM16) and found some improvements. colour could also profit from it. Check out https://arxiv.org/pdf/1802.06067.pdf?fname=cm&font=TypeI, in particular page 3, or its implementation in colorio https://github.com/nschloe/colorio/blob/master/colorio/ciecam02.py.

**Comment by thomasmansencal** [comment]
That looks great @nschloe! I think it would be worth contacting the authors about it. We are usually not integrating anything that is not coming from _official_ literature though but will definitely keep an eye on it.

**Comment by nschloe** [comment]
Yeah well I _am_ the author. Just a quick write-up right there.

**Comment by thomasmansencal** [comment]
Oh I was referring to the CAM16 authors :)

**Comment by nschloe** [comment]
Ah yes. Okay, let's try to get officially endorsed. :)

**Comment by thomasmansencal** [comment]
That would be great!

**Comment by thomasmansencal** [comment]
@nschloe : Any news on this front?

**Comment by nschloe** [comment]
Uh... What exactly do you have in mind? I wrote the authors and got a replay back from Changjun Li (in Feb 2018) saying that they'll have a look. That's fair enough, right? It doesn't really matter if the authors look at it, right? The note is more for anyone who wants to implement CIECAM02/CAM16.

**Comment by thomasmansencal** [comment]
Fair enough! Your last comment was:

> Ah yes. Okay, let's try to get officially endorsed. :)

So I was just inquiring if there was any progress.

**Comment by nschloe** [comment]
Yeah, I mean, it'd be nice if this was used by upstream since the standard is a bit flawed here and there, but I cannot _make_ anyone use it. At least the ideas are out there for everyone to implement.

And besides, I'm working on a color model that will perhaps obsolete all of CAM ones anyway. :smile: 

https://github.com/nschloe/colorio/commit/eedda66bf06826ba656969fd979516df9aac9069

**Comment by thomasmansencal** [comment]
Ah! :)

Thanks, I will close this one to tidy up our stack a bit!

**Comment by nschloe** [comment]
I wouldn't close this before having made the respective changes, e.g., [here](https://github.com/colour-science/colour/blob/a3e7182d6a991beecd7162dd8ba891b556d79732/colour/appearance/ciecam02.py#L847).
